### PR TITLE
Add  document-question-answering in task_summary

### PR DESCRIPTION
**Pull request by y3sar**
# What does this PR do?

From issue #18926
This PR adds Document Question Answering summary to task_summary.mdx
It also provides an example using pipeline and this [model](https://huggingface.co/naver-clova-ix/donut-base-finetuned-docvqa)  

## Who can review?
@stevhliu

### CODE REVIEW

File: docs/source/en/task_summary.mdx [hunk 1]
```diff
@@ -235,6 +235,27 @@ There are two common types of question answering:
 score: 0.9327, start: 30, end: 54, answer: huggingface/transformers
 ```
 
+### Docuement Question answering
```
Comment (stevhliu) [inline docs/source/en/task_summary.mdx:238]:
```suggestion
### Document question answering
```

File: docs/source/en/task_summary.mdx [hunk 2]
```diff
@@ -235,6 +235,27 @@ There are two common types of question answering:
...
+
+Document Question answering is a task that answers natural language questions from a document. Unlike token-level question answering task which takes text as input, document question answering takes an image of a document as input along with a question about the document and returns an answer. Document Question answering can be used to parse structured documents and extract key information from documents. For example total amount and change amount can be extracted from a receipt.
```
Comment (stevhliu) [inline docs/source/en/task_summary.mdx:240]:
```suggestion
Document question answering is a task that answers natural language questions from a document. Unlike a token-level question answering task which takes text as input, document question answering takes an image of a document as input along with a question about the document and returns an answer. Document question answering can be used to parse structured documents and extract key information from documents. In the example below, the total amount and change due can be extracted from a receipt.
```

**Review by stevhliu — COMMENTED** [review]
Very nice, thanks for taking this up! 🤗 

This is more of a multimodal task, so I think it'd be better to create a new `## Multimodal` section for this task (kind of like [this](https://moon-ci-docs.huggingface.co/docs/transformers/pr_23252/en/task_summary#natural-language-processing)).

**Comment by y3sar** [comment]
@stevhliu I only changed the docs. I don't know why the checks keep failing. It says it is having problems with ffmpeg?

File: docs/source/en/task_summary.mdx [hunk 3]
```diff
@@ -307,4 +307,29 @@ There are two types of language modeling:
       'sequence': 'Hugging Face is a community-based open-source platform for machine learning.'}]
     ```
 
+## Multimodal
+
+Multimodal tasks require a model to process multiple data modalities (text, image, audio, video) to solve a particular problem. A simple example of a multimodal task could be image captioning where the model takes an image as input and outputs a sequence of text describing the image or some properties of the image. Although multimodal models seem to work with different data types internally the preprocessing steps help the model convert all the data types into a similar form called embeddings (vectors or list of numbers that holds meaningful information about the data). So the model learns relationships between sequence of patches (small chunks of images) of image embeddings and text embeddings to complete a task like image captioning.
```
Comment (stevhliu) [inline docs/source/en/task_summary.mdx:312]:
```suggestion
Multimodal tasks require a model to process multiple data modalities (text, image, audio, video) to solve a particular problem. Image captioning is an example of a multimodal task where the model takes an image as input and outputs a sequence of text describing the image or some properties of the image. 

Although multimodal models work with different data types, internally, the preprocessing steps help the model convert all the data types into embeddings (vectors or list of numbers that holds meaningful information about the data). For a task like image captioning, the model learns relationships between image embeddings and text embeddings.
```

**Review by stevhliu — COMMENTED** [review]
I left a suggestion to simplify the multimodal section a bit and to break up the text so it is more readable. Otherwise, looks good to me!

Looks like your [`merge`](https://github.com/huggingface/transformers/pull/23252/commits/41038273c4242f39b47142fe631d45281261c430) may have introduced some other changes. I think maybe closing this and opening a new PR would the easiest here? What do you recommend @amyeroberts?

**Comment by y3sar** [comment]
Thank you so much for you help @stevhliu I think I should open a new PR. I made a mess here

**Comment by y3sar** [comment]
I created a new pull request #23318 closing this PR

**Pull request closed without merging**